format_strawberryfield.routing.yml

```diff
@@ -231,3 +231,18 @@ format_strawberryfield.deletetmp_webannotations:
     _format: 'json'
     _entity_access: 'node.update'
     _permission: 'add strawberryfield webannotation'
+
+# Display settings for each ADO
```

Comment:
Tiny thing. These are Display settings for bundles of type ADO, not for each ADO right?